### PR TITLE
Add functionality to Bottom Bar

### LINKED ISSUE #113: Implement Bottom Bar
**What are the tasks that need to be done?**
- [x] Add chat box to bottom bar
- [x] Add Day/Night cycle tracker to bottom bar

### In-Depth Feature Description
The BottomBar component is currently empty and should contain some elements as per mockups. 

### Images for Reference:
image

### Feature Approval:
- [x]  Feature approved by: @ateagit (Alan Thomas)

### PR DESCRIPTION
**Pull request by skyn310198**
# Description
This PR fixes #113

+Add functionality to the Bottom Bar - be able to track Day/Night of the game
-Exclude chat box - future implementation

# Demo
After:
image

image

# Type of change:
* New feature (Non-breaking change that adds functionality)

### CODE REVIEW

File: frontend/src/Components/BottomBar.jsx [hunk 1]
```diff
@@ -1,10 +1,21 @@
 import { React } from 'react';
+import PropTypes from 'prop-types';
 // import styles from '../Styles/BottomBar.module.css';
 
-const BottomBar = () => (
+const BottomBar = ({Period, Number}) => (
     <div>
```
Comment (ateagit) [inline frontend/src/Components/BottomBar.jsx:6]: Not really a problem but you can access period and number directly here (so you dont have to pass props) by using the context
`const { state } = useContext(GameContext)` (check the StatusBar.js component for an example)

File: frontend/src/Socket.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import socketIOClient from 'socket.io-client';
 
-const socket = new socketIOClient('mafia-hands-backend-cd.herokuapp.com');
+// const socket = new socketIOClient('mafia-hands-backend-cd.herokuapp.com');
```
Comment (ateagit) [inline frontend/src/Socket.js:3]: Remember to revert this file to use heroku when making a pr

**Review by ateagit — CHANGES_REQUESTED** [review]
Good stuff

**Review by jteo97 — APPROVED** [review]
Everything looks good to me.

**Review by ateagit — APPROVED** [review]
LGTM

**Review by jge385 — APPROVED** [review]
Good job~ Looks good to me

**Comment by SiyuQian** [comment]
Good stuff! Thanks.

**Review by SiyuQian — APPROVED** [review]

**Pull request merged**